src/amr/tagging/tagger.h

```diff
@@ -2,8 +2,8 @@
 #ifndef PHARE_TAGGER_H
 #define PHARE_TAGGER_H
 
-#include "physical_models/physical_model.h"
 #include "amr/types/amr_types.h"
+#include "solver/physical_models/physical_model.h"
```

Comment:
typically, solver depends on AMR

Reply:
also, adding in "#include "solver" in amr creates a bi-directional dependency

Reply:
yeah that's not good.
But since I created this solver lib, I have disliked it.... I created it with in mind that it would not depend on SAMRAI explicitly, but it does through the use of the PatchHierarchy, PatchLevels and Patches... I've almost reached the point where I seriously consider moving all this code back to amr...

Reply:
on the other hand, there is code that's in amr, namely some tagging stuff, or some refinement and coarsening stuff, that do not depend on SAMRAI and that could be better located in core... TBC...